__tests__/app.test.js

```diff
@@ -29,16 +29,41 @@ describe("app", () => {
           .get("/api/topics/")
           .expect(200)
           .then(({ body }) => {
-            expect(body).toEqual([
-              { slug: "mitch", description: "The man, the Mitch, the legend" },
-              { slug: "cats", description: "Not dogs" },
-              { slug: "paper", description: "what books are made of" },
-            ]);
+            expect(typeof body).toBe("object");
             body.forEach((topic) => {
               expect(Object.keys(topic)).toEqual(["slug", "description"]);
             });
           });
       });
     });
+    describe("GET /api/articles/:article_id", () => {
+      test("Status 200: Should respond with an article object using the id", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then((res) => {
+            expect(typeof res.body).toBe("object");
+          });
+      });
```

Comment:
You need to be testing for alot more on the response here, just checking it's an object isn't enough and doesn't give me/us/yourself much confidence of what's come back. For example the response will indeed be an object but how are you checking the key/value pairs within it? The request is for article id number 1, so your assertions here should be that checking that you have got the exact expected article back for id 1.

Request: Please can you extend this test, add more confidence and then **re-submit** asap.